stdlib/Cargo.toml

```diff
@@ -19,3 +19,4 @@ crate-type = ["staticlib", "rlib"]
 [features]
 default = []
 range-support = ["llvm-tools"]
+no-rt-fail = []
```

Comment:
Guidance is that features should be additive, not subtract functionality
https://doc.rust-lang.org/cargo/reference/features.html#feature-unification

Reply:
Yeah, I agree. I think we should mimic the pattern we used for `range-support` and make it a additive feature like `fail-support` and then include it in qir-runner and qir-backend.

Reply:
Agree, should we make it a default feature then?

Reply:
Yeah, I think that makes the most sense. Range-support is not default because it's a weird one that only affects linking with Q# generated QIR and brings in additional build time tool dependencies and compatibility issues. The fail-support doesn't have that baggage so can more reasonably be part of default.

Reply:
Implemented the feature as `fail-support` which is a default feature.